finetune/adapter_v2.py

```diff
@@ -169,7 +183,11 @@ def validate(fabric: L.Fabric, model: torch.nn.Module, val_data: np.ndarray, tok
     prompt = generate_prompt(sample)
     encoded = tokenizer.encode(prompt, device=model.device)
     output = generate(
-        model, idx=encoded, max_returned_tokens=len(encoded) + 100, max_seq_length=max_seq_length, temperature=0.8
+        model,
+        idx=encoded,
+        max_returned_tokens=len(encoded) + 100,
+        max_seq_length=max_seq_length,
```

Comment:
```suggestion
        max_seq_length=model.config.block_size,
```

Reply:
Maybe this is more of a conceptual question, but why was there a `max_seq_length` in the fine tuning scripts in the first place if you can always just use the model block size? 

Like, let's say the fine-tuning data is prepared with a smaller `max_seq_length`, would you want to also propagate that to the fine-tuning logic or is actually better to just always use the block size?

Reply:
Related question, if the fine-tuning scripts don't actually need max_seq_length then it seems fine to change the config loading to `config = Config.from_name(name=checkpoint_dir.name)`from  `config = Config.from_name(name=checkpoint_dir.name, block_size=max_seq_length)` but I want to make sure I'm not misunderstanding how things work.